crossing-training.lic

```diff
@@ -413,13 +413,22 @@ class CrossingTraining
   end
 
   def research
+    if @settings.cyclic_research
+      research_spell = @settings.crossing_cyclic_research_spells([])
```

Comment:
This appears to be a list of skills, and I feel 'crossing' is not communicating any information. I recommend naming it `cyclic_research_skills`.